Healthy/Classes/Modules/MainTabBarController/MainTabBarController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    
+    private(set) var tabBarAddButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.backgroundColor = .primary100
+        button.setImage(UIImage(systemName: "plus"), for: .normal)
+        button.tintColor = .white
+        button.layer.cornerRadius = 28
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTabBar()
+        configureVieControllers()
+    }
+
+    private func configureTabBar() {
+        setupMainTabBar()
```

Comment:
```suggestion
        configureMainTabBar()
```
Following the convention.